modules/trusted-firmware-m/Kconfig

```diff
@@ -239,19 +256,28 @@ config TFM_PS_VALIDATE_METADATA_FROM_FLASH
 
 config TFM_PS_MAX_ASSET_SIZE
 	int "Maximum stored asset size"
+	range 512 4024
 	default 2048
 	help
-	  The maximum asset size to be stored in the Protected Storage area
+	  The maximum asset size to be stored in the Protected Storage area. The maximum value
+	  is limited by the memory block size, which must also hold the encryption header.
+	  The actual size of the stored asset is not affected by this configuration.
 
 config TFM_PS_NUM_ASSETS
 	int "Maximum stored assets number"
+	range 1 120
 	default 10
 	help
-	  The maximum number of assets to be stored in the Protected Storage
-	  area
+	  The maximum number of assets to be stored in the Protected Storage area.
+	  (TF-M does not guarantee that the Protected Storage area is large
+	  enough to store the maximum amount of assets with the maximum size.) Set
```

Comment:
```suggestion
	  enough to store the maximum amount of assets with the maximum size) Set
```
nit
Or you can even just remove the parenthesis all together.

Reply:
No, this is incorrect. Keep the parenthesis and the period.